docs/architecture/adr-005-handlers-redesign.md

```diff
@@ -296,7 +303,7 @@ trait ValidationContext {
     /// Function required by ICS 03. Returns the list of all possible versions that the connection
     /// handshake protocol supports.
     fn get_compatible_versions(&self) -> Vec<Version> {
-        get_compatible_versions()
+        ConnectionVersion::compatibles()
```

Comment:
Thanks for updating the ADR as well. Can you please revert the markdown formatting?